RealtekHD.txt

```diff
@@ -0,0 +1,17 @@
+[Section]
+Name = Realtek HD Audio Drivers R2.74 (WDM)
+Version = R2.74 (5.10.0.7111)
+License = Propietary
+Description = Realtek HD Audio codec and drivers, suitable for multiple soundcards/chipsets and devices as workstation, laptops, minipc, etc. 
```

Comment:
optional. Notice the part with the EOL-whitespace-stripping. That part is mandatory.
```suggestion
Description = Realtek HD Audio codec and drivers, suitable for multiple soundcards/chipsets.
```